### PR TITLE
spell check pass

### PR DESCRIPTION
**Pull request by sbrkopac**
## Description

Spell check pass

### CODE REVIEW

**Comment by robertosfield** [comment]
Wow.  Many thanks for all these fixes.

Deteriorating eye sight and possible Dyslexia are a killer typo combo...

File: src/vsg/state/GraphicsPipelineStates.cpp [hunk 1]
```diff
@@ -242,7 +242,7 @@ void ViewportState::read(Input& input)
         {
             input.read("x", scissor.offset.x);
             input.read("y", scissor.offset.y);
-            input.read("wdith", scissor.extent.width);
+            input.read("width", scissor.extent.width);
```
Comment (sbrkopac) [inline src/vsg/state/GraphicsPipelineStates.cpp:245]: Possible breaking change.

File: src/vsg/state/GraphicsPipelineStates.cpp [hunk 2]
```diff
@@ -270,7 +270,7 @@ void ViewportState::write(Output& output) const
         {
             output.write("x", scissor.offset.x);
             output.write("y", scissor.offset.y);
-            output.write("wdith", scissor.extent.width);
+            output.write("width", scissor.extent.width);
```
Comment (sbrkopac) [inline src/vsg/state/GraphicsPipelineStates.cpp:273]: Possible breaking change.

**Review by sbrkopac — COMMENTED** [review]
I was doing a quick review of this again and forgot to mention that there was a possible breaking change. I normally just test against my own project for VSG which doesn't include any provided binary formats. Just wanted to note that if there are serialized binaries for the Viewport state, we will have an issue.

**Pull request merged**